touchforms/backend/xformplayer.py

```diff
@@ -322,11 +322,9 @@ def ix_in_scope(form_ix):
             evt = self.__parse_event(form_ix)
             evt['relevant'] = relevant
             if evt['type'] == 'sub-group':
-                presentation_group = (evt['caption'] != None)
-                if presentation_group:
-                    siblings.append(evt)
-                    evt['children'] = []
-                form_ix = self._walk(form_ix, evt['children'] if presentation_group else siblings)
+                siblings.append(evt)
+                evt['children'] = []
+                form_ix = self._walk(form_ix, evt['children'])
```

Comment:
I stared at this for a while and can't figure out what cases are different and how. do you have a summary? like why you want want to call `self._walk` with `siblings` and whether it's a problem that that no longer happens...